Core/Object Arts/Dolphin/MVP/Base/OS.SCROLLINFO.cls

```diff
@@ -35,7 +35,7 @@ fMask: anInteger
 	bytes dwordAtOffset: _OffsetOf_fMask put: anInteger!
 
 maskIn: anInteger
-	"OR in to contents' mask member the bit flags represented by anInteger.
+	"OR in to content's mask member the bit flags represented by anInteger.
```

Comment:
OR in to the mask field the bit flags ...

Reply:
How about ...the receiver's mask field the bit flags... ?

Reply:
I'm not especially bothered whether it is "OR into the mask field" or "OR into the receiver's mask field"